networks/rpc/http.go

```diff
@@ -240,11 +241,12 @@ func NewFastHTTPServer(cors []string, vhosts []string, timeouts HTTPTimeouts, sr
 
 	// TODO-Klaytn concurreny default (256 * 1024), goroutine limit (8192)
 	return &fasthttp.Server{
-		Concurrency:  ConcurrencyLimit,
-		Handler:      fhandler,
-		ReadTimeout:  timeouts.ReadTimeout,
-		WriteTimeout: timeouts.WriteTimeout,
-		IdleTimeout:  timeouts.IdleTimeout,
+		Concurrency:        ConcurrencyLimit,
+		Handler:            fhandler,
+		ReadTimeout:        timeouts.ReadTimeout,
+		WriteTimeout:       timeouts.WriteTimeout,
+		IdleTimeout:        timeouts.IdleTimeout,
+		MaxRequestBodySize: common.MaxRequestContentLength,
```

Comment:
I have a minor question...
```
func validateFastRequest(requestCtx *fasthttp.RequestCtx) (int, error) {
        ...
	if requestCtx.Request.Header.ContentLength() > common.MaxRequestContentLength {
		err := fmt.Errorf("content length too large (%d>%d)", requestCtx.Request.Header.ContentLength(), common.MaxRequestContentLength)
		return http.StatusRequestEntityTooLarge, err
	}
```
Is it right to check here?
If so, I wonder why the `defaultMaxRequestBodySize` is used without this change.

Reply:
@hyochan-brown  There are two kinds of data size restrictions in our code. 
1) `validateFastRequest` is an application side logic with a default max length 512 KB. The restriction size is configurable.
2)  `MaxRequestBodySize` introduced in this PR is an HTTP server (library) side logic with a default size 4MB. The restrictions size was not configurable, but will be configurable with this PR. 

Commonly. an application side restriction is more strict than http server side restriction. Therfore, the HTTP server side restriction was not important. However, the server side restriction is effective when a user configure the application side restriction size with a large value (more than 4MB). 
Without this change, a user cannot input more than 4MB data even though he/she configure `MaxRequestContentLength` with a large value.

Reply:
I totally understood.
Thank you so much. :)